src/main/java/frc/robot/commands/climber/Climb.java

```diff
@@ -8,24 +8,20 @@
 package frc.robot.commands.climber;
 
 import edu.wpi.first.wpilibj.DoubleSolenoid.Value;
-import edu.wpi.first.wpilibj.Joystick;
 import edu.wpi.first.wpilibj2.command.CommandBase;
-import frc.robot.Constants;
 import frc.robot.subsystems.Climber;
 
 public class Climb extends CommandBase {
 
   private Climber m_climber;
-  private Joystick m_stick;
-  private int m_axis1, m_axis2;
+  private double m_speed;
 
   /**
    * Creates a new Climb.
    */
-  public Climb(Joystick stick, int axis1, int axis2, Climber climber) {
+  public Climb(double speed, Climber climber) {
```

Comment:
instead of a `double` here, I think you want a `DoubleSupplier` which is really just an interface with a `DoubleSupplier.get()` method that returns a `double`. The intent here is to use a lambda to get the input speed for the climber from the joysticks, not to replace joystick input with a constant speed.